mmv1/third_party/terraform/tests/resource_storage_bucket_object_test.go

```diff
@@ -18,6 +18,7 @@ import (
 
 const (
 	objectName = "tf-gce-test"
+	folderName = "tf-gce-folder-test/"
```

Comment:
Could this be scoped to the function it's used it? 

```golang
func TestAccStorageObject_folder(t *testing.T) {
```

Reply:
done